### PR TITLE
Bugfix: update the value mapping with the correct result of graphop.

### PR DESCRIPTION
**Pull request by bgogul**
(1) Fixes SR-8298.
(2) Updates the test to run the frontend with strict deabstraction turned on and off. 

Some tests did not have any CHECKs. I have left it as is for now.

### CODE REVIEW

**Review by lattner — APPROVED** [review]
whoops, nice fix thanks!

File: lib/SILOptimizer/Mandatory/TFLowerGraph.cpp [hunk 1]
```diff
@@ -2119,7 +2119,7 @@ GLStatus TFGraphLowering::visitGraphOperationInst(GraphOperationInst *inst) {
 
   // Remember each of the results.
   for (unsigned i = 0; i != numOpResults; ++i) {
-    addValueMapping({inst->getResult(0), 0}, {result, (int)i});
+    addValueMapping({inst->getResult(i), 0}, {result, (int)i});
```
Comment (mhong) [inline lib/SILOptimizer/Mandatory/TFLowerGraph.cpp]: nice!

after completing the migration to graph op inst, which natively supports multiple result values, my understanding is that we can remove the second component `0` in `{inst->getResult(i), 0}` -- is that correct?

**Comment by mhong** [comment]
I believe no_copy.swift is intended to not have CHECKs for each test -- instead its main focus is to make sure the test cases do not generate any "copy to host" / "copy to accel" warnings.

File: lib/SILOptimizer/Mandatory/TFLowerGraph.cpp (hunk 1 above)
Comment (lattner) [reply lib/SILOptimizer/Mandatory/TFLowerGraph.cpp]: yes, exactly right!  I look forward to that day.

**Pull request merged**